src/components/playerOverview/PlayerDetails.tsx

```diff
@@ -53,27 +62,44 @@ export const PlayerDetails = React.memo(function PlayerDetails() {
         }
     }
 
+    const colorsPlayedArray = [
```

Comment:
as it stands, atm this implementation is too fragile. if we accidentally change the order of colorsPlayedArray and/or the order of MTG_COLOR_NAMES or MTG_COLOR_RGB, we'll have problems. can we combine them into a single array of "MagicColor" or "CommanderColor" objects that have 3 properties, "name", "id", and "value" (where value is the RGB value)

Then when we need it, we can use MtgColors.map((colorObj) => ....)) to scope down to a an array of color values, color names, etc. This way the order is consistent and guaranteed.